connector/connector.py

```diff
@@ -422,6 +422,9 @@ def to_openerp(self, external_id, unwrap=False):
             bindings = getattr(bindings, self._openerp_field)
         return bindings
 
+    # 2016-10-25: compatible with old implementations
+    to_openerp = to_odoo
```

Comment:
As said, it can be done the other way. @sergio-teruel, please change it and take care about the overrides.